### ISSUE TITLE
sr_subscribe list redirection is empty...

### ISSUE THREAD

**Issue by petersilva** [MetPX/sarracenia#296]
```

blacklab% ./sr_subscribe.py list  

configuration examples: ( /home/peter/src/sarracenia/sarra/examples/subscribe )
WMO_Sketch_2mqtt.conf  WMO_Sketch_2v3.conf    WMO_mesh_CMC.conf   WMO_mesh_Peer.conf        amqp_f30.conf      aws-nexrad.conf      cclean_f91.conf       cdnld_f21.conf 
      cfile_f44.conf          cp_f61.conf        dd_2mqtt.conf          dd_all.conf         dd_amis.conf         dd_aqhi.conf dd_cacn_bulletins.conf     dd_citypage.conf 
        dd_cmml.conf         dd_gdps.conf         dd_ping.conf        dd_radar.conf         dd_rdps.conf         dd_swob.conf     ddc_cap-xml.conf      ddc_normal.conf 
  downloademail.conf      ec_ninjo-a.conf         ftp_f70.conf       hpfx_amis.conf       local_sub.conf    pitcher_pull.conf           q_f71.conf  rabbitmqtt_f31.conf 
         sci2ec.conf         subnoaa.conf      subsoapshc.conf         subusgs.conf      u_sftp_f60.conf 


general: ( /home/peter/.config/sarra )
          admin.conf     credentials.conf         default.conf 


user configurations: ( /home/peter/.config/sarra/subscribe )
       amqp_f30.conf      cclean_f91.conf       cdnld_f21.conf       cfile_f44.conf          cp_f61.conf         ftp_f70.conf           q_f71.conf  rabbitmqtt_f31.conf 
     u_sftp_f60.conf 

logs are in: /home/peter/.cache/sarra/log

blacklab%
```
but if you try to filter or re-direct it, you get nothing:
```

sr_subscribe list >kk
blacklab% ls -al kk
-rw-r--r-- 1 peter peter 0 Jan  3 19:44 kk
blacklab% 

```

**Comment by petersilva** [comment]
patch fixes this: 0f35a01e7adf060f3d6213f5075dd5525407a92e

**Comment by petersilva** [comment]
released in v2.20.01b3
